docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -0,0 +1,110 @@
+|===
+|Name | Default | Description
+
+|spring.cloud.gcp.bigquery.credentials.default-scopes |  | 
+|spring.cloud.gcp.bigquery.enabled | true | Auto-configure Google Cloud BigQuery components.
+|spring.cloud.gcp.bigquery.project-id |  | 
+|spring.cloud.gcp.config.credentials.default-scopes |  | 
+|spring.cloud.gcp.config.enabled | false | Enables Spring Cloud GCP Config.
+|spring.cloud.gcp.config.name |  | Name of the application.
+|spring.cloud.gcp.config.profile |  | Comma-delimited string of profiles under which the app is running. Gets its default value from the {@code spring.profiles.active} property, falling back on the {@code spring.profiles.default} property.
```

Comment:
Don't we need to be as simple as possible in the configuration properties descriptions?
I mean we should avoid to use any JavaDocs tag: we can't be sure what  tool is going to render such a description.
So, tags like this (or that `<code>`) below could break the target tool.